pyproject.toml

```diff
@@ -35,6 +35,9 @@ dependencies = [
     "packaging>=23.0.0",                     # https://github.com/pypa/pip-audit/issues/464
     "pip-api>=0.0.28",
     "pip-requirements-parser>=32.0.0",
+    # NOTE(ww): requests 2.30.0 and higher are incompatible with CacheControl;
```

Comment:
This comment isn't quite correct, it's urllib3 v2.0 that isn't compatible with CacheControl: https://github.com/psf/requests/issues/6437#issuecomment-1533357246

Reply:
Thanks! Will fix.